apps/studio/components/interfaces/Billing/NoProjectsOnPaidOrgInfo.tsx

```diff
@@ -28,11 +28,13 @@ export const NoProjectsOnPaidOrgInfo = ({ organization }: NoProjectsOnPaidOrgInf
       type="default"
       title={`Your organization is on the ${orgSubscription.plan.name} plan with no projects running`}
       description={
-        <Markdown
-          className="!max-w-full"
-          content={`The monthly fees for the paid plan still apply. To cancel your subscription, head over to
-          your [organization billing settings](/dashboard/org/${organization?.slug}/billing).`}
-        />
+        <div className="!max-w-full prose text-sm">
+          The monthly fees for the paid plan still apply. To cancel your subscription, head over to
+          your{' '}
+          <Link href={`/dashboard/org/${organization?.slug}/billing`}>
```

Comment:
Now we're outside of markdown we don't need to prepend the `/dashboard/` ourselves.
```suggestion
          <Link href={`/org/${organization?.slug}/billing`}>
```